app/experimenter/targeting/constants.py

```diff
@@ -825,6 +825,17 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TEST_STICKY_TARGETING = NimbusTargetingConfig(
+    name="Test targeting",
+    slug="test_targeting",
+    description="Config for sticky targeting",
+    targeting="'sticky.targeting.test.pref'|preferenceValue",
+    desktop_telemetry="",
+    sticky_required=False,
```

Comment:
Why wouldn't this be `sticky_required=True`?

Reply:
So in the test I set the sticky field itself, but seeing as this is a sticky targeting pref, it should be set here too I guess. I can change it

Reply:
Ok sweet. I wasn't sure if that would be a separate test or something 👍